packages/http-client-java/generator/http-client-generator-core/src/main/java/com/microsoft/typespec/http/client/generator/core/extension/plugin/JavaSettings.java

```diff
@@ -218,7 +218,7 @@ private JavaSettings(AutorestSettings autorestSettings) {
         this.syncMethods = SyncMethodsGeneration.fromValue(getStringValue(host, "sync-methods", "essential"));
 
         // Whether to add a logger to the generated clients.
-        this.clientLogger = getBooleanValue(host, "client-logger", false);
+        this.clientLogger = getBooleanValue(host, "client-logger", true);
```

Comment:
This may affect vanilla. AFAIK vanilla mode does not include ClientLogger in generated code.

Reply:
Adding ClientLogger should not be an issue. This should have no public API impact and also, logging exceptions should be okay for the vanilla code gen.

Reply:
Got it.

Just loop @alzimmermsft for awareness.